django/applications/catmaid/control/sampler.py

```diff
@@ -1,7 +1,9 @@
-import json
+#!/usr/bin/env python
```

Comment:
Why the shebang?

Reply:
If you don't want it there that's fine - I normally add it to all python code to make it very clear to code editors that this file is python.

Reply:
I see, thanks for the explanation. While I understand your point, I am in favor of not having it there for the sake of consistency. Having editors recognize this is a Python file hasn't been a problem so far I believe.